llvm/lib/CodeGen/SelectionDAG/SelectionDAGBuilder.cpp

```diff
@@ -3330,7 +3330,7 @@ void SelectionDAGBuilder::visitInvoke(const InvokeInst &I) {
           EHPadMBB->setMachineBlockAddressTaken();
       break;
     case Intrinsic::experimental_patchpoint_void:
```

Comment:
Do we still need a specific void intrinsic?

Reply:
I tried removing it, and it seems like it changes the mangled name to `@llvm.experimental.patchpoint.isVoid`. I'm not sure if this would be desireable.

Reply:
The mangling for void is the ugly isVoid, but I don't see that as a great reason to leave around an intrinsic that only differs by adding a concrete return type

Reply:
Should I open another PR with that change?